### PR TITLE
Notifications redirect

### PR DESCRIPTION
**Pull request by MarcusBoay**
Related #161 

Turns out this was an easy implementation on the notifications side. However, the Activity that the notification spawns depends on the Request's status via intent extras. Shouldn't the request status be obtained within the Activity itself?

### CODE REVIEW

File: app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java [hunk 1]
```diff
@@ -96,11 +96,14 @@ public void onChanged(@Nullable List<Notification> notifications) {
...
-//        Intent showBookIntent = new Intent(getApplicationContext(), /* TODO: WAITING ON REFACTORED BOOKINFOACTIVITY */);
-//        showBookIntent.putExtra("bookID", notification.getBookID());
-//        startActivity(showBookIntent);
-//    }
+    private void showBookInfo(Notification notification) {
+        Intent showBookIntent = new Intent(getApplicationContext(), ShowRequestInfoActivity.class);
+        showBookIntent.putExtra("bookID", notification.getBookID());
+        // TODO: rStatus in ShowRequestInfoActivity should be obtained within itself
+        // BELOW IS PLACEHOLDER AND SHOULD BE THE STATUS OF THE REQUEST INSTEAD OF THE NOTIFICATION
+        showBookIntent.putExtra("rStatus", notification.getrNotificationType().toString());
```
Comment (AY-Chen) [inline app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java]: Might be good to make rStatus a static final constant in the target class(ShowRequestInfoActivity) and then reference that constant instead

**Review by AY-Chen — COMMENTED** [review]
It works! Only one small change to use a constant instead if you can. 

I thought about the actual UI flow and maybe calling finish on the previous activity, but that gets into some sketchy territory because you might end up calling finish on the MainActivity or something you don't intend. There's some merit to just opening the new activity like that. If we were to change the flow, I'd suggest setting the previous activity to wherever we decide to hold and show all the requests, and then open the new activity. Not sure though, we really should add showing all the requests somewhere first.

File: app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java (hunk 1 above)
Comment (MarcusBoay) [reply app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java]: I'm not really sure about everything that's going on with the Activity currently. Do you think I should leave it now and refactor in the future or?

Comment (AY-Chen) [reply app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java]: I think just change it now. It should be fairly easy to change

Comment (MarcusBoay) [reply app/src/main/java/com/example/atheneum/activities/NotificationsActivity.java]: done

**Review by ishchow — APPROVED** [review]

**Pull request merged**